dev_scripts/merge_large_tests_results.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
```

Comment:
Should we try to rely on already existing tools for this, or what we do is somehow too specific? A quick lookup on the PyPI index got me this: https://pypi.org/project/junitparser/

Reply:
D'oh, I didn't even check for that :facepalm: . Great find, I'll incorporate it in this dev script.